### ISSUE TITLE
Measure: bug send many concurrent requests for no reason adminkml and profile

### ISSUE THREAD

**Issue by loicgasser** [geoadmin/mf-geoadmin3#4105]
This explains a lot of downtimes on the alti service.

Probably a problem with event registration

**Steps to reproduce**

1.  Open mapgeoadmin and measure tool
2.  Draw a line with 4 points
3. Deselect line 

**Result**

A profile request is sent

**Expected result**

No profile request is sent

----------------------------------------------------------------------------------

**New steps**

1. reselect

Result

2 profiles requests are sent

**New steps**

1. deselect

Result

2 profiles requests are sent

**New steps**

1. reselect 

Result¨

3 profiles requests are sent

and so on

--------------------------------------------

Basically each time the user reselect drwaing there is one additional request triggered in parallel.

----------------------------------------------

I also propose to change the debounce to 2 sec.

**Comment by oterral** [comment]
There was multiple problems:

- POST request are not cached by Angular
- Event on feature's change was not removed correctly when the profile's scope was destroyed
- Feature's change event was triggered on each change of feature's style (selection/deselection)
